src/components/Records/Record/GeneralInfo/ReplacedByRecord.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div
+    v-if="(!!currentRecord['fairsharingRecord'].reverseRecordAssociations.find(item=>item.recordAssocLabel==='deprecates') && currentRecord['fairsharingRecord'].reverseRecordAssociations) || (!!currentRecord['fairsharingRecord'].recordAssociations.find(item=>item.recordAssocLabel==='deprecates' && currentRecord['fairsharingRecord'].recordAssociations))"
```

Comment:
This if statement is fairly complex and could be hard to understand when it comes to making changes to this code later.
Though it will mean adding an extra test, it would be worthwhile abstracting this calculation into a separate method. The method could then be commented to describe what it does, e.g.

```
<div
    v-if="replacedOrReplaces"
    ...
>
```

```
methods: {
    // Return true if this record deprecates another or is deprecated by another. 
    replacedOrReplaces: () => {
        // find relations in (reverse) associations here and return a boolean.
    }
}
```

The same could possibly be applied to other v-ifs here, but this one is the most important as I think it will be the hardest for future maintainers to deal with.